templates/createrepo-update.sh.erb

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+<%-
+    command = @createrepo_update
+-%>
+
+if [ "$(whoami)" != '<%= @repo_owner %>' ]; then
+    echo "You really should be '<%= @repo_owner %>' while running '$0'"
+    exit 1
+fi
+
+<%- if @use_filter -%>
+[ $# -eq 0 ] || echo "$1" | grep -q '\.rpm$' || exit
+
+<%- end -%>
+
+<%- if @use_lockfile -%>
+find "<%= @lockfile %>" -cmin +1500 -exec rmdir '{}' +
+mkdir "<%= @lockfile %>" || exit
+
+trap "rmdir <%= @lockfile %>; exit" INT TERM EXIT
```

Comment:
I see you migrated away from using `flock`. Did that cause issues?

Reply:
This is because our rpms reside on an nfs share; and flock is not available there.